orders/views.py

```diff
@@ -1,21 +1,41 @@
-import json
+import json, re
```

Comment:
```suggestion
import json
```
현재 PR에서 사용하지 않는 import 같습니다.